source/classes.tex

```diff
@@ -1089,14 +1085,18 @@
 \end{example}
 
 \pnum
+\begin{note}
 Similarly, \tcode{volatile} semantics\iref{dcl.type.cv} apply in
 volatile member functions when accessing the object and its
 non-static data members.
+\end{note}
 
 \pnum
-A cv-qualified member function can be called on an
-object-expression\iref{expr.ref} only if the object-expression is as
-cv-qualified or less-cv-qualified than the member function.
+A member function whose type has a \grammarterm{cv-qualifier-seq} \cvqual{cv1}
+can be called on an
+object expression\iref{expr.ref} of type \cvqual{cv2} \tcode{T} only
+if \cvqual{cv2} is not more cv-qualified than
+\cvqual{cv1}\iref{basic.type.qualifier}.
```

Comment:
"not more" is not right. You can't call a `volatile` member function on a `const` object, even though "volatile" is not more cv-qualified than "const" (or vice-versa - they are unordered).

Reply:
Fixed.